run_benchmark.sh

```
@@ -2,50 +2,143 @@
+  TIMING_ENTRIES+=("{\"mode\":\"$mode\",\"tps\":$BENCHMARK_TARGET_TPS,\"total_requests\":$BENCHMARK_COUNT_VAR,\"status\":\"$result\",\"prometheus_timestamps\":{\"start_ms\":$start,\"finish_ms\":$end}}")
+}
+
+run_signature() {
+  source envs/client.env.sh
+  export CLIENT_PAYMENT_MODE="signature"
+  export CLIENT_PAYMENT_COUNT=$BENCHMARK_COUNT_VAR
+  run_mode "signature"
+}
+
+run_paytree() {
+  source envs/client.env.sh
```

Shellcheck (0.11.0)

[info] Not following: envs/client.env.sh was not specified as input (see shellcheck -x).

(SC1091)